src/main/java/io/vertx/redis/client/impl/types/IntegerType.java

```diff
@@ -34,6 +34,11 @@ private IntegerType(long value) {
   public ResponseType type() {
     return ResponseType.INTEGER;
   }
+  
+  @Override
+  public Integer toInteger() {
+    return value.intValue();
```

Comment:
RFC: do we want to report somehow when the value is out of range for `int`?

* use `assert` so it only throws when assertions are enabled
* throw `IllegalStateException`
* log an error/warning

Any of these checks would impose overhead but not nearly as much as round-tripping through a string.

Some users may also intend to perform a truncating cast so maybe we should only return the value.